### PR TITLE
Update docs for limit_predict_batches

### PR DESCRIPTION
**Pull request by rohitgr7**
## What does this PR do?

Fixes https://github.com/PyTorchLightning/pytorch-lightning/issues/6490 + some minor updates.

## Before submitting
- [x] Was this discussed/approved via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your PR does only one thing, instead of bundling different changes together?
- [x] Did you make sure to update the documentation with your changes? (if necessary)
- [ ] Did you write any new necessary tests? (not for typos and docs)
- [ ] Did you verify new and existing tests pass locally with your changes?
- [ ] Did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)? (not for typos, docs, test updates, or internal minor changes/refactorings)

## PR review
Anyone in the community is free to review the PR once the tests have passed.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:

 - [x] Is this pull request ready for review? (if not, please submit in draft mode)
 - [x] Check that all items from **Before submitting** are resolved
 - [x] Make sure the title is self-explanatory and the description concisely explains the PR
 - [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

File: pytorch_lightning/trainer/trainer.py [hunk 1]
```diff
@@ -204,6 +204,8 @@ def __init__(
 
             limit_test_batches: How much of test dataset to check (floats = percent, int = num_batches)
 
+            limit_predict_batches: How much of prediction dataset to check (floats = percent, int = num_batches)
```
Comment (kaushikb11) [inline pytorch_lightning/trainer/trainer.py:207]:
```suggestion
            limit_predict_batches: How much of prediction dataset to check (float = percent, int = num_batches)
```

Comment (awaelchli) [reply pytorch_lightning/trainer/trainer.py:207]: we need to say float = fraction, it can't be percent because the value is between 0 and 1

Comment (rohitgr7) [reply pytorch_lightning/trainer/trainer.py]: Updated.

**Review by awaelchli — APPROVED** [review]

**Review by carmocca — APPROVED** [review]

**Review by kaushikb11 — APPROVED** [review]

**Review by tchaton — APPROVED** [review]
Nice !

**Pull request merged**